config/settings.yml.example

```diff
@@ -58,6 +58,11 @@
 # default values for https_port is 8443
 #:https_port: 8443
 
+# http server type configuration
```

Comment:
Is this setting needed? AFAIK Rails prefers puma but falls back when it's unavailable. I think it would also work well for the proxy.

Reply:
Not sure what you mean, there's no Rails here. We want Puma to be opt-in only for the first phase.

Reply:
Rails automatically uses Puma when it's available but silently falls back to webrick. I'm suggesting to copy that behavior for the proxy but I can be convinced to be explicit.

Reply:
Scenario: Puma is installed, a bug is found that prevents your production deployment from operation. How do you switch back to webrick? Uninstallation is not intuitive. Another scenario would be try in puma, switch over to webrick to retest and then back and forth. This is where configuration shines.

Reply:
:+1: for choosing explicitly